draft-kuehlewind-update-tag.md

```diff
@@ -182,12 +182,14 @@ probably qualify to have one of the new tags defined in this document.
 
 ## Formating Style of Amendments
 
-This document does not impose any requirements on the form of the amendment
-made. Some RFCs use and OLD/NEW style to highlight actual text changes others
-simply describe the changes in text. Both can make sense in certain situation.
-However, this document does recommend to use the OLD/NEW rather for smaller and
-a limited number of changes, while if larger or many changes are needed, a new
-document revision that obsoletes the old RFC should be considered.
+Currently some RFCs use and OLD/NEW style to highlight actual text changes others
+simply describe the changes in text. This document recommends the use of the 
```

Comment:
```suggestion
simply describe the changes in text. While this document does no require a specific format of amendments,
it recommends the use of the 
```

Reply:
Looks good. Replace "does no" with "does not"